ReplicationDemo.ps1

```diff
@@ -0,0 +1,99 @@
+# dbatools 💜 dbatools
```

Comment:
This should be incorporated into the devcontainer

Reply:
Yep, good shout - will add a TODO and get that done - might move to an issue so it's outside of this PR but will see how far I get this week.

Reply:
I'm wondering if maybe we just create a dedicated demo repository for this? Then that repo could have a dedicated devcontainer setup to load into VS code for users.

Reply:
Make it easier if we do this in a dedicated repo just use docker-compose that way users can see how we are building this out and play with adjusting as they want. I can help convert your docker run commands into docker-compose if you need.

We have [dataplat/dbatools-lab](https://github.com/dataplat/dbatools-lab) for the book, so could make say a `dataplat/dbatools-repl-lab` repo.

Reply:
Was this file going to be removed/moved to a new repo?